README.md

```diff
@@ -116,12 +116,39 @@ Use this instead of `MVMT` in future steps.
 ```
 conda create -n MVMT python=3.9 -y
 conda activate MVMT
+```
+
+You could run into issues when installing dependencies. Check that your executable permissions are valid by running python and python3.9
```

Comment:
This should only be an issue for internal collaborators working on `/ofo-share`. Can you mention this to avoid confusing external users?

Reply:
Good point, I'll add the clarification.